cfn-resources/online-archive/cmd/resource/resource.go

```diff
@@ -361,18 +330,25 @@ func validateProgress(ctx context.Context, client *mongodbatlas.Client, currentM
 	p := handler.NewProgressEvent()
 	p.OperationStatus = cloudformation.OperationStatusSuccess
 	p.Message = "Complete"
-	if archive.State != "DELETED" {
+	if *archive.State != "DELETED" {
```

Comment:
should DELETE be a constant?

Reply:
i changed to be used only one place so i suppose it's not some important now